tests/unit/stream_alert_rule_processor/test_handler.py

```diff
@@ -360,3 +360,11 @@ def put_record_batch(**kwargs):
 
         self.__sa_handler.run(test_event)
         assert_true(mock_logging.error.called)
+
+    @patch('stream_alert.rule_processor.handler.StreamRules.load_intelligence')
```

Comment:
Side note: we need to figure out a better story with end-to-end unit tests.  We only use a single config ('/test/unit/conf') in our test cases, however that doesn't represent all the possible combinations in the real world, which affects how logic is forked.